cloudaeye/README.md

```diff
@@ -0,0 +1,32 @@
+# CloudAEye
+
+## Overview
+
+Kosal automates root cause analysis (RCA), significantly improving incident resolution time and streamlining the experience for engineers. This AI-powered workflow replaces manual tasks that engineers currently handle by analyzing data - logs, metrics, traces - and pin-pointing the root cause with relevant context. As a result, engineers can redirect their focus to more critical aspects. Additionally, Kosal adeptly identifies historical issues within Jira and traces related code changes that may lie at the heart of the problem.
+
+## Setup
+
+### Installation
+
+Visit [CloudAEye][4] to sign up for free. Once registered, visit the [CloudAEye integrations page][1] and add a Datadog integration. This guides you through the Datadog OAUTH2 flow to grant CloudAEye access to your traces, logs, and metrics data.
```

Comment:
OAuth needs to be intiated within Datadog. Please update the steps users will take to set up the integration